pkg/operator/ceph/cluster/controller.go

```diff
@@ -218,15 +222,17 @@ func add(mgr manager.Manager, r reconcile.Reconciler, context *clusterd.Context)
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileCephCluster) Reconcile(context context.Context, request reconcile.Request) (reconcile.Result, error) {
 	// workaround because the rook logging mechanism is not compatible with the controller-runtime logging interface
-	reconcileResponse, err := r.reconcile(request)
+	reconcileResponse, cephCluster, err := r.reconcile(request)
+
 	if err != nil {
 		logger.Errorf("failed to reconcile. %v", err)
+		r.clusterController.recorder.Report(cephCluster, cephCluster.Name, "Warning", "FailedReconcile", err.Error())
```

Comment:
How about we also record an event for successful if `err == nil`?

Reply:
In case of success, what should we record? `Reconcile succeed`